server/JsDbg.Gdb/GdbDebugger.cs

```diff
@@ -531,7 +531,9 @@ class GdbDebugger : IDebugger {
             string response = await this.QueryDebuggerPython("GetCurrentProcessThreads()");
             return ParsePythonArrayToIntegers(response).ToArray();
         }
-        public async Task<ulong> TebAddress() { return 0; }
+
+        public Task<ulong> TebAddress() { return Task.FromResult(0UL); }

```

Comment:
Not related to this change...TEB is a Windows-only concept, so it doesn't make sense to show this extension on other platforms. I think we should have a declarative way to write platform-specific extensions. Perhaps we can add a "targetPlatforms" field to the extension manifest? I'll file an issue to track this.

Reply:
I believe the TEB extension is already limited to the "ntdll" module

Reply:
That's true, forgot about that...
So it isn't a problem in this case. But I think this is applicable to the WinD3D11AngleTextures extension, and other extensions we may right for platform-specific technologies in the future. The TSF extension (not checked-in, but there's a branch for it) is another example. Doesn't make sense to have these on Linux/Mac.

Reply:
Yeah, I agree.

Reply:
Filed issue #86 .